### ISSUE TITLE
How to implement gradient accumulation based on oneflow.optimizer

### ISSUE THREAD

**Issue by GrittyChen** [Oneflow-Inc/oneflow#3811]
Gradient accumulation is a widely used method to achieve large batch size training with limited GPU memory. Could you provide the document about how to implement gradient accumulation based on oneflow.optimizer (or any customized optimizer)? Thanks very much!

**Comment by MARD1NO** [comment]
Thanks for you feedback. Now the gradient accumulation is developing, it will merge into master soon. 

If you still have other questions, please comment below~

**Comment by GrittyChen** [comment]
> Thanks for you feedback. Now the gradient accumulation is developing, it will merge into master soon.
> 
> If you still have other questions, please comment below~

Thanks for your prompt reply. I am looking forward to the update.

**Comment by yuanms2** [comment]
gradient accumulation is merged
